backend/infrahub/events/branch_action.py

```diff
@@ -24,6 +25,8 @@ def get_resource(self) -> dict[str, str]:
 
     def get_messages(self) -> list[InfrahubMessage]:
         events = [
+            # TODO: Sending EventBranchDelete currently has no effect.
```

Comment:
Let's keep it for now, once all the events have been migrated we'll change the behavior of the queue in the message bus